oscarlab/graphene
# Remove trailing blanks

## Affected components


## Description of the changes (reasons and measures)
This is the first patch in a series of patches cleaning up and unifying the formatting of our codebase, which will be finished with a patch that enables and enforces `clang-format` checks on Jenkins (see #539).

After we merge this PR we should stop accepting trailing blanks in reviews (though it won't yet be enforced in CI).

I tried to change only files which are part of our codebase and was already adapted by us, so e.g. shim/test/apps directory is mostly skipped.

## How to test this PR? (if applicable)
Run `grep -Ir '[[:blank:]]$' .` and see if there are any more occurrences that I should also include.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/664)
